test/integration/capacity.test.ts

```diff
@@ -109,6 +109,33 @@ describe('capacity', () => {
       )
     })
 
+    it('it should get all transactions from a capacity ID - 200', async () => {
+      const response = await get(app, `/capacity/${seededCapacityId}/creation/`)
+      expect(response.status).to.equal(200)
+      expect(response.body).to.deep.equal(
+        [
+          {
+            id: '1f3af974-7d4d-40b4-86a5-94a2241265cb',
```

Comment:
you can use the exported `seededTransactionId`s for these IDs, also a few enums instead of things like `DEMAND`